packages/agreement-process/src/services/readModelService.ts

```diff
@@ -562,6 +611,69 @@ export function readModelServiceBuilder(
         ),
       };
     },
+    async getDelegationByDelegateId(
```

Comment:
This seems to be unused

Reply:
It should be used by [assertRequesterIsConsumerOrProducerOrDelegateProducer](https://github.com/pagopa/interop-be-monorepo/blob/618942e8fc1a076ba5c6db1b9b4c767c12d2a582/packages/agreement-process/src/model/domain/agreement-validators.ts#L175)